mobile_menu.js

```diff
@@ -0,0 +1,17 @@
+const menu = document.getElementById('menu2');
+const cancel = document.getElementById('cancel');
+const nav = document.getElementById('menu');
+
+function Show() {
+  nav.classList.add('Show');
+  cancel.style.display = 'flex';
+  menu.style.display = 'none';
+}
+
+menu.addEventListener('click', Show);
+function Hide() {
+  nav.classList.remove('Show');
+  cancel.style.display = 'none';
+  menu.style.display = 'flex';
+}
+cancel.addEventListener('click', Hide);
```

Comment:
- Please, kindly ensure that when a user clicks on the menu list items, the mobile menu should not be visible as this part is not yet implemented in your script.